lib/sequenceserver/routes.rb

```diff
@@ -108,6 +108,17 @@ class Routes < Sinatra::Base
       sequences.to_json
     end
 
+    post '/get_sequence' do
+      sequences = Sequence::Retriever.new(params["sequence_ids"].split(","),
+                                          params["database_ids"],
+                                          params["download"])
+
+      send_file(sequences.file.path,
+                :type     => sequences.mime,
+                :filename => sequences.filename) if params["download"]
+      sequences.to_json
```

Comment:
@yeban ok removed. I just wan't sure if there wasn't some other functionality that used this.